compliance/azure/ahub_manual/README.md

```diff
@@ -1,57 +1,38 @@
-  - `AZURE_APPLICATION_ID`
-  - `AZURE_APPLICATION_KEY`
-  
-### Required RightScale Roles
+## Input Parameters
 
-- policy_designer
-- policy_manager
-- policy_publisher
-- Cloud Management - credential_viewer
-
-### Azure Required Permissions
-
-- Microsoft.Compute/virtualMachines/read
-- Microsoft.Compute/locations/vmSizes/read
+This policy has the following input parameters required when launching the policy.
 
-### Installation
+- *Email addresses to notify* - Email addresses of the recipients you wish to notify when new incidents are created
+- *Allowed AHUB licenses* - Number of AHUB licenses that are allowed to be run on Azure
+- *Exclusion Tag Key* - Azure VMs instance tag to ignore instance that are with AHUB enabled. Only supply the tag key. The policy assumes that the tag value is irrelevant.
 
-1. Follow steps to [Create an Azure Active Directory Application](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#create-an-azure-active-directory-application)
-1. Grant the Azure AD Application access to the necessary subscription(s)
-1. [Retrieve the Application ID & Authentication Key](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-application-id-and-authentication-key)
-1. Create RightScale Credentials with values that match the Application ID (Credential name: `AZURE_APPLICATION_ID`) & Authentication Key (Credential name: `AZURE_APPLICATION_KEY`)
-1. [Retrieve your Tenant ID](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-tenant-id)
+## Prerequesites
 
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud -- in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is compatible with this policy. The information below should be consulted when creating the credential.
 
-### Input Parameters
+### Azure Required Permissions
 
-This policy has the following input parameters required when launching the policy.
+The following Azure permissions must be allowed for the policy to run.
 
-- *Azure AD Tenant ID* - the Azure AD Tenant ID used for the Azure API Authentication
-- *Azure Subscription ID* - the Azure Subscription ID used for the Azure API Authentication
-- *Email addresses to notify* - Email addresses of the recipients you wish to notify when new incidents are created
-- *Allowed AHUB licenses* - Number of AHUB licenses that are allowed to be run on Azure
-- *Exclusion Tag Key* - Azure VMs instance tag to ignore instance that are with AHUB enabled. Only supply the tag key. The policy assumes that the tag value is irrelevant.
+- Microsoft.Compute/virtualMachines/read
```

Comment:
Are there other permissions needed for AHUB?

Reply:
There is no such API used that may need specific AHUB permissions.